README.md

```diff
@@ -63,6 +63,8 @@ This repository is the official implementation of BLAKE3. It includes:
 
 * [![Actions Status](https://github.com/BLAKE3-team/BLAKE3/workflows/tests/badge.svg)](https://github.com/BLAKE3-team/BLAKE3/actions)
 
+* [![TrustInSoft CI](https://ci.trust-in-soft.com/projects/BLAKE3-team/BLAKE3.svg?branch=master)](https://ci.trust-in-soft.com/projects/BLAKE3-team/BLAKE3)
```

Comment:
I have a slight preference for merging this line with the one above, to keep the testing badges together.

Reply:
Done!